lib/config/config.go

```diff
@@ -374,6 +377,11 @@ func (cfg *Configuration) clean() error {
 	return nil
 }
 
+func convertV27V28(cfg *Configuration) {
```

Comment:
I have a counter suggestion. There is now a generic metadata kv-store area in the database, look in main for where it checks for the upgrade detection. We could have an integer there which is the database schema version or something and avoid doing this based on the config version. Pro: it avoids the "hack" on what is not really a config migration. Con: it's one more magic integer that causes stuff to happen. Discuss.

Reply:
I am in favor of one (hopefully) more magic integer with some semantic connection to the stuff it causes to happen instead of one number used for anything.